topics/go/algorithms/strings/permuation/permutation.go

```diff
@@ -1,27 +1,25 @@
 package strings
 
 import (
+	strings "github.com/ardanlabs/gotraining/topics/go/algorithms/strings/types"
```

Comment:
Don't do this. Keep everything together in this single file. It doesn't matter if we need it again.

Reply:
Ok, Thanks.